middlewares/validators/recruiter.js

```diff
@@ -0,0 +1,27 @@
+const joi = require('joi')
+
+const validateRecruiter = async (req, res, next) => {
+  const schema = joi.object().keys({
+    company: joi.string().optional(),
+    first_name: joi.string().optional(),
+    last_name: joi.string().optional(),
+    designation: joi.string().optional(),
+    reason: joi.string().optional(),
+    email: joi.string().optional(),
+    currency: joi.string().optional(),
+    package: joi.number().optional(),
+    timestamp: joi.string().optional()
```

Comment:
Are we getting `timestamp` from frontend? Backend is generating that right? Please correct if I am wrong. 😃

Reply:
It is like we need the time at which the Recruiter Contacted a member.
This will help us to add in the `Notification` of member.

Reply:
What she meant was, do we really need to validate on `timestamp`? It won't be coming in from `req.body`, right?

We should NOT be sending timestamp from frontend (if we are, just in case). Any user could send random timestamp (of future or past) and that would get stored in db, which is not something we want.

Reply:
@Rucha1499 Sorry it is added mistakenly.
It is not coming through req (from frontend) ,we are later adding it in the DB.

Reply:
> What she meant was, do we really need to validate on `timestamp`? It won't be coming in from `req.body`, right?
> 
> We should NOT be sending timestamp from frontend (if we are, just in case). Any user could send random timestamp (of future or past) and that would get stored in db, which is not something we want.

My Bad 😅